### PR TITLE
Move tools no SQL to module

### PR DESCRIPTION
**Pull request by tungtt22**
- [x] Jenkins
- [x] Nat Instance
- [x] Squid
- [x] Nexus
- [x] Bastion
- [x] Prometheus
- [x] Grafana

### CODE REVIEW

File: terraform/main.tf [hunk 1]
```diff
@@ -19,10 +30,67 @@ resource "aws_vpc" "devops" {
   }
 }
 
+# Define Bastion inside the public subnet
```
Comment (locvx1234) [inline terraform/main.tf:33]: why don't you use the module structure?

File: terraform/modules/nat_instance/main.tf [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  }
+
+  tags = {
+    Name = var.project_name != "" ? "${var.project_name}-NAT-Server" : "NAT-Server"
+  }
+
+  volume_tags = {
+    Name = var.project_name != "" ? "${var.project_name}-NAT-Server" : "NAT-Server"
+  }
+}
```
Comment (locvx1234) [inline terraform/modules/nat_instance/main.tf]: put an end line pls =))

File: terraform/modules/nexus/main.tf [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  }
+
+  tags = {
+    Name = var.project_name != "" ? "${var.project_name}-Nexus-Server" : "Nexus-Server"
+  }
+
+  volume_tags = {
+    Name = var.project_name != "" ? "${var.project_name}-Nexus-Server" : "Nexus-Server"
+  }
+}
```
Comment (locvx1234) [inline terraform/modules/nexus/main.tf]: endline again \n

File: terraform/modules/squid/main.tf [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  }
+
+  tags = {
+    Name = var.project_name != "" ? "${var.project_name}-Squid-Server" : "Squid-Server"
+  }
+
+  volume_tags = {
+    Name = var.project_name != "" ? "${var.project_name}-Squid-Server" : "Squid-Server"
+  }
+}
```
Comment (locvx1234) [inline terraform/modules/squid/main.tf]: /n

File: terraform/main.tf (hunk 1 above)
Comment (tungtt22) [reply terraform/main.tf:33]: it's not special tool then no need to move

**Comment by tungtt22** [comment]
@locvx1234  Please help to review again

**Review by locvx1234 — APPROVED** [review]
🥇

**Pull request merged**